modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json

```diff
@@ -252,7 +252,9 @@
       "TOBIRA_PATH_SEGMENT_INVALID": "Pfadsegment ist ungültig. Ein gültiges Pfadsegment ist mindestens zwei Zeichen lang, enthält keine Leerzeichen, Kontrollzeichen, keins der Zeichen '<>\"[\\]^`{|}#%\/?]' und fängt mit keinem der Zeichen '-+~@_!$&;:.,=*'()' an.",
       "TOBIRA_PATH_SEGMENT_UNIQUE": "Das gleiche Pfadsegment darf nicht mehrmals für Seiten mit der gleichen Überseite verwendet werden.",
       "TOBIRA_PAGE_NOT_FOUND": "Die angefragte Seite existiert nicht (mehr). Versuchen Sie wieder vom Anfang zu beginnen.",
-      "TOBIRA_SERVER_ERROR":" Fehler bei der Kommunikation mit Tobira. Kontaktieren Sie Ihren Systemadministrator oder versuchen Sie es später erneut."
+      "TOBIRA_SERVER_ERROR":" Fehler bei der Kommunikation mit Tobira. Kontaktieren Sie Ihren Systemadministrator oder versuchen Sie es später erneut.",
+      "TOBIRA_COPIED_DIRECT_LINK": "Verknüpfung wurde erfolgreich in die Zwischenablage kopiert.",
+      "TOBIRA_FAILED_COPYING_DIRECT_LINK": "Verknüpfung konnte nicht in die Zwischenablage kopiert werden."
```

Comment:
Why "Verknüpfung" and not simply "link"? It's also called "Direktlink" elsewhere.

Reply:
I was feeling overly German, I guess. x)